FlowCrypt/Functionality/Mail Provider/Message Provider/Gmail+MessageExtension.swift

```diff
@@ -148,12 +148,12 @@ private extension GTLRGmail_Message {
 
     func parseAttachments() -> [MessageAttachment] {
         attachmentParts.compactMap {
```

Comment:
probably separate `attachmentParts` property is not needed anymore, as it just takes original `payload.parts` array, and can be replaced directly with `payload?.parts?.compactMap { ... } ?? []`